docs/intro/quickstart/modeling.rst

```diff
@@ -0,0 +1,101 @@
+.. _ref_quickstart_modeling:
+
+=================
+Modeling the data
+=================
+
+.. edb:split-section::
+
+  The flashcards application has a simple data model, but it's interesting enough to get a taste of many of the features of the |Gel| schema language. You have a ``Card`` type that describes a single flashcard, which for now contains two required string properties: ``front`` and ``back``. Each ``Card`` belongs to a ``Deck``, and there is an explicit ordering to the cards in a given deck.
```

Comment:
It looks like there's no `order` in the TypeScript version

Reply:
Right: it's just the order of the array itself. That's why I didn't mention an `order` property, just said "there _is_ an explicit ordering". I'll maybe make this more clear that I mean arrays are ordered and we want to keep this order in the database.